generators/typescript/sdk/client-class-generator/src/endpoints/default/endpoint-response/GeneratedThrowingEndpointResponse.ts

```diff
@@ -772,7 +772,12 @@ export class GeneratedThrowingEndpointResponse implements GeneratedEndpointRespo
                         ),
                         [
                             ts.factory.createThrowStatement(
-                                context.timeoutSdkError.getGeneratedTimeoutSdkError().build(context)
+                                context.timeoutSdkError
+                                    .getGeneratedTimeoutSdkError()
+                                    .build(
+                                        context,
+                                        `Timeout exception generated in endpoint ${this.endpoint.name.camelCase.unsafeName}.`
```

Comment:
We might actually want to include the full endpoint name representation (e.g. `POST /users/{userId}`). You can use a combination of the endpoint method and [this function](https://github.com/fern-api/fern/blob/cf894f08ab4d3f0dad59f4512a3c822ac558530e/generators/typescript/sdk/generator/src/SdkGenerator.ts#L1074) to determine that. 

Otherwise, it'll be too difficult to know what endpoint actually timed out (e.g. `list` could be `user.list` or `organization.list`, etc).

Reply:
From slack +1, ideal message would be `Timeout of 60s exceeded in POST /users/v1` but the timeout is configurable

Reply:
Great, implemented!